src/js/carousel.js

```diff
@@ -0,0 +1,33 @@
+const carousel = document.querySelector(".carousel");
```

Comment:
tip: use $carousel or carouselEl to know when a variable references to a DOM element.